compiler/testData/codegen/box/delegatedProperty/optimizedDelegatedProperties/kt48825.kt

```diff
@@ -0,0 +1,25 @@
+// IGNORE_BACKEND: JVM
+// MODULE: lib
+// FILE: lib.kt
+import kotlin.reflect.KProperty
+
+open class A {
+    inline operator fun Int.getValue(thisRef: Any?, property: KProperty<*>): String =
+        property.name
+}
+
+// MODULE: main(lib)
+// FILE: main.kt
+import kotlin.reflect.KProperty
+
+open class B : A() {
+    inline operator fun String.getValue(thisRef: Any?, property: KProperty<*>): String =
+        property.name
+}
+
+class C : B() {
+    val O by 1
+    val K by ""
+}
+
+fun box() = C().O + C().K
```

Comment:
In #4592, I've noticed that the test also checks `setValue`... Could you add it here please?

Reply:
done